DataFormats/TrackerCommon/src/TrackerTopology.cc

```diff
@@ -85,6 +85,29 @@ unsigned int TrackerTopology::module(const DetId &id) const {
   return 0;
 }
 
+std::pair<int,int> TrackerTopology::stripSubDetAndLayer(const DetId& id) const
```

Comment:
I personally do not like a method called "Strip..." in TrackerTopology. TrackerTopology has no idea whether a subdetector is a SiStrip subdetector or and phase 2 OT subdetector or a pixel subdetector. I see two ways to improve. 
The simple one: a method calle subDetAndLayer in TrackerTopology which returns the subdet (from 1 to 6) and which returns TrackerTopology::layer 
A more complicated one: a method called stripSubDetAndLayer in TrackerGeometry which returns the subdet and the layer ONLY if the detid belongs to a subdetector which, in that geometry is a SiStrip subdetector.